db/migrate/20180404153832_add_otp_enabled_to_accounts.rb

```diff
@@ -0,0 +1,5 @@
+class AddOtpEnabledToAccounts < ActiveRecord::Migration[5.1]
+  def change
+    add_column :accounts, :otp_enabled, :boolean, default: false
```

Comment:
here you can use syntax like `add_column :accounts, :otp_enabled, :boolean, after: :column_name`